src/app/space/create/deployments/deployments-toolbar/deployments-toolbar.component.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnInit,
+  Output,
+  SimpleChanges,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  FilterConfig,
+  FilterEvent,
+  FilterField,
+  FilterType,
+  SortConfig,
+  SortEvent,
+  ToolbarConfig
+} from 'patternfly-ng';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'deployments-toolbar',
+  templateUrl: './deployments-toolbar.component.html'
+})
+export class DeploymentsToolbarComponent implements OnChanges, OnInit {
+
+  public filterConfig: FilterConfig;
+  public isAscendingSort: boolean = true;
+
+  @Output('onFilterChange') public onFilterChange: EventEmitter<FilterEvent> = new EventEmitter<FilterEvent>();
+  @Output('onSortChange') public onSortChange: EventEmitter<SortEvent> = new EventEmitter<SortEvent>();
+
+  @Input() public resultsCount: number;
+
+  private sortConfig: SortConfig;
+  private toolbarConfig: ToolbarConfig;
+
+  public constructor() { }
+
+  public filterChange($event: FilterEvent): void {
+    this.onFilterChange.emit($event);
+  }
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    if (changes.resultsCount && this.filterConfig) {
+      this.filterConfig.resultsCount = changes.resultsCount.currentValue;
+    }
+  }
+
+  public ngOnInit(): void {
+    this.filterConfig = {
+      appliedFilters: [],
+      fields: [{
+        id: 'applicationId',
```

Comment:
Could you extract this ID to a public const and reference that over in the deployments-apps.component, so that there isn't a duplicated magic string across the two files?